ngraph/test/CMakeLists.txt

```diff
@@ -412,13 +411,8 @@ if(NGRAPH_ADDRESS_SANITIZER)
 endif()
 
 target_link_libraries(unit-test PRIVATE ngraph_test_util
-                                        openvino::conditional_compilation
-                                        openvino::itt
-                                        openvino::pp
-                                        ngraph
-                                        ngraph::builder
-                                        ngraph_backend
-                                        libgtest)
+                                        ngraph::builder)
+ie_mark_target_as_cc(unit-test)
```

Comment:
do we need it? I suppose tests are not conditionally compiled.

Reply:
Yes we need it, because [cc tests](https://github.com/openvinotoolkit/openvino/tree/master/ngraph/test/conditional_compilation) include header which requires `conditional_compilation` library.

Reply:
for such cases I think we can link CC library directly. Because this cmake macro also adds objects dependency from CC header.. Maybe it's not needed for tests

Reply:
Agree. I linked CC library directly.